src/translations/en/translations.json

```diff
@@ -36,12 +84,17 @@
   "is not supported": "is not supported",
   "No transactions found!": "No transactions found!",
   "The section will show you recent transactions history after first payment": "The section will show you recent transactions history after first payment",
-  "We’re sorry to see you go.": "We’re sorry to see you go.",
+  "We’re sorry to see you go": "We’re sorry to see you go",
+  "Your <1>{{offerTitle}}</1>": "Your <1>{{offerTitle}}</1>",
+  "free trial will end on ": "free trial will end on ",
+  "subscription is paid until ": "subscription is paid until ",
+  "<0>{{formattedExpiresAt}}</0>. If you would like to proceed with cancelling your subscription, please select 'Unsubscribe' below, and your subscription will be cancelled as of <2>{{formattedExpiresAt}}</2>. Until then, you will continue to have access to all of your current subscription features. Before you go, please let us know why you're leaving.": "<0>{{formattedExpiresAt}}</0>. If you would like to proceed with cancelling your subscription, please select 'Unsubscribe' below, and your subscription will be cancelled as of <2>{{formattedExpiresAt}}</2>. Until then, you will continue to have access to all of your current subscription features. Before you go, please let us know why you're leaving.",
```

Comment:
Yes. That's because we have to use that Trans component because the previous solution doesn't work. Here is a link for documentation https://react.i18next.com/latest/trans-component#using-with-react-components
I think that in the future we should make everything to make texts simpler and not use that solution because it makes code a little ugly.